nautobot/docs/development/getting-started.md

```diff
@@ -178,7 +178,23 @@ Additional useful commands for the development environment:
 - `invoke stop` - Stops all containers created by `invoke start`
 
 !!! info
-    By default the Nautobot docker images for development are built with Python 3.7. If you wish to build them with a different Python version, stop any running containers, then set the environment variable `PYTHON_VER` to the desired version (for example, `export PYTHON_VER=3.8`) and rerun the `invoke build` command. As long as `PYTHON_VER` remains set, all other `invoke` tasks will use the containers built for this version instead of the default.
+    By default the Nautobot docker images for development are built with Python 3.7. If you wish to build them with a different Python version, stop any running containers, then set the environment variable `INVOKE_NAUTOBOT_PYTHON_VER` to the desired version (for example, `export INVOKE_NAUTOBOT_PYTHON_VER=3.8`) and rerun the `invoke build` command. As long as `INVOKE_NAUTOBOT_PYTHON_VER` remains set, all other `invoke` tasks will use the containers built for this version instead of the default. See the [invoke configuration](#invoke-configuration) for more information.
+
+#### Invoke Configuration
+
+The invoke tasks have some default configuration set via Invoke's [Configuration](http://docs.pyinvoke.org/en/stable/concepts/configuration.html#environment-variables) which you may want to override, these include:
+
+- `python_ver`: the python version which is used to build the docker container (default: 3.7)
+- `local`: run the commands in the local environment vs the Docker container (default: False)
+- `compose_dir`: the full path to the directory containing the docker-compose yaml files (default: "/full/path/to/nautobot/development")
+- `compose_file`: the docker-compose yaml tile to use (default: "docker-compose.yml")
+- `compose_override_file`: the default docker-compose override file to use if it exists (default: "docker-compose.override.yml")
+
+These setting may be overridden several different ways (listed in order of precedence):
+
+- Command line argument on the individual commands (see `invoke $command --help`) if available
+- Using environment variables such as `INVOKE_NAUTOBOT_PYTHON_VER` the variables are prefixed with `INVOKE_NAUTOBOT_` and must be uppercase
```

Comment:
Worth spelling out how to set boolean variables (like `INVOKE_NAUTOBOT_LOCAL`)? I.e. must it be `INVOKE_NAUTOBOT_LOCAL=True`, or does say `INVOKE_NAUTOBOT_LOCAL=1` also work?